# PWM control for left and right motors
int8 left
int8 right